cypress/integration/gift_card_spec.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+import { HomePo } from '../support/home.po';
+import { GiftCardsPo } from '../support/giftCard.po';
+
+
+describe('100$ gift card add to cart', () => {
+    const homePage = new HomePo();
+    const giftPage = new GiftCardsPo();
+
+    beforeEach(() => cy.visit(homePage.pageUrl));
+
+    it('gift card add to cart', () => {
+
+        const successMessage = 'Added to Cart';
+        cy.viewport(1175, 600);
+
+        giftPage.clickOnElem(giftPage.giftCardLink);
+        giftPage.isURLcontain(giftPage.giftCardUrl);
+        giftPage.clickOnElem(giftPage.eGift);
+        giftPage.isElemVisible(giftPage.prise100);
+        giftPage.clickOnElem(giftPage.prise100);
+        giftPage.typeIntoElement(giftPage.emailField, 'test@gmail.com');
+        giftPage.isElemContainText(giftPage.totalPrice, '$100');
+        giftPage.clickOnElem(giftPage.addToCartButton);
+        giftPage.isElemContainText(giftPage.addedToCart, successMessage);
```

Comment:
here, at the finished point, you can add as many checks, as you want to be sure that really this specific product was added:
cy.get('#huc-v2-order-row-inner')
.should('to.contain', 'Added to Cart')
.and('to.contain', 'Proceed to checkout')
.and('to.contain', 'Cart subtotal')
.and('to.have.descendants', 'img');